style.css

```diff
@@ -1,9 +1,76 @@
+body {
+    width: 1440px;
+}
 
-.header{
-    display:flex;
-    flex-direction: row;
+.header {
+    display: flex;
+    height: 30px;
+    width: 1440px;
 }
-.nav{
-    display:flex;
+
+.nav {
+    display: flex;
     flex-direction: row;
+
+    justify-content: space-between;
+    width: 100%;
+
+}
+
+.link_flex_container {
+    display: flex;
+    justify-content: flex-end;
+    width: 80%;
+
+}
+.logo{
+    font-size: 25px;
+}
+.links {
+    margin: 0px 10px 0px 10px;
+    text-decoration: none;
+    color: black;
+    text-transform: capitalize;
+    font-size: 25px;
+}
+
+.sign {
+    text-decoration: none;
+    color: black;
+    text-transform: capitalize;
+    padding-right: 115px;
+    font-size: 25px;
+}
+
+.first_page {
+    display: flex;
+    height: 900px;
+}
+
+.main_photo {
+    height: 436px;
+    width: 609px;
```

Comment:
good, you should set exact size on photos

Reply:
Ty :)